app/plex_tools.py

```diff
@@ -167,17 +180,17 @@ def search_plex():
         else:
             print("| Config Error: {} method not supported".format(method))
     elif plex.library_type == "show":
-        if method == "tmdb_list" and TMDB.valid and TraktClient.valid:
+        if method == "tmdb_list":
             shows, missing = imdb_tools.tmdb_get_shows(config_path, plex, value, is_list=True)
-        elif method in ["tmdb_id", "tmdb_show"] and TMDB.valid and TraktClient.valid:
```

Comment:
We don't need these validations anymore?

Reply:
It was a not needed second check they would never fail because it would raise an error before then if one of those is needed but not valid